src/metrics.js

```diff
@@ -45,6 +45,51 @@ const metrics = [
     docs: "https://taskman.eionet.europa.eu/projects/netpub/wiki/Quality_metrics#Seo-Score",
     help: "SEO Score - evaluated using evaluated using <a href='https://developers.google.com/web/tools/lighthouse' target='_blank'>Lighthouse</a> in Chrome Developer Tools (Audits)"
   },
+  {
+    name: "Performance (Fast Connection)",
+    database: "lighthouse",
+    measurement: "performance-score_fast",
+    field: "value",
+    thresholds: [50, 90],
+    docs: "https://taskman.eionet.europa.eu/projects/netpub/wiki/Quality_metrics#Performance",
+    help: "Performance - evaluated using <a href='https://developers.google.com/web/tools/lighthouse' target='_blank'>Lighthouse</a> in Chrome Developer Tools"
+  },
+  {
+    name: "Progressive Web App (Fast Connection)",
```

Comment:
The scores for Progressive Web App, Accessibility, Best Practice & SEO should mostly remain the same, regardless of connection speed. There is no need to add these measurements too; only Performance (Fast) is needed.